examples/echo_server.ts

```diff
@@ -2,8 +2,9 @@
 const { listen, copy } = Deno;
 
 (async (): Promise<void> => {
```

Comment:
I think you can remove this anonymous function too

Reply:
Done